docs/index.md

```diff
@@ -28,7 +28,7 @@ This tool is intended for use by developers who are experienced with JavaScript.
 ## Including the Buy SDK
 
 ```html
-<script src="http://sdks.shopifycdn.com/js-buy-sdk/latest/shopify-buy.polyfilled.globals.min.js"></script>
+<script src="http://sdks.shopifycdn.com/js-buy-sdk/v0/latest/shopify-buy.umd.polyfilled.min.js"></script>
```

Comment:
Not sure what you mean by your last sentence there, but I definitely think there is a use case for providing a stable API, that can incorporate progressive bug fixes. We have explicitly versioned APIs for those that want that, but this is the same argument of wether or not we pin all our versions, or fuzzy match. In the best case scenario, we support both approaches. The cool thing about this is that most people are familiar with fuzzy matching against major version, and it's far less risky (especially for those who have opted in with `latest` without understanding the risk).